src/cpptools.ts

```diff
@@ -28,7 +28,10 @@ export class CppConfigurationProvider implements cpp.CustomConfigurationProvider
     private workspaceBrowseConfiguration: cpp.WorkspaceBrowseConfiguration = { browsePath: [] };
 
     private getConfiguration(uri: vscode.Uri): SourceFileConfigurationItem | undefined {
-        const norm_path: string = path.normalize(uri.fsPath);
+        let norm_path: string = path.normalize(uri.fsPath);
+        if (process.platform === "win32") {
+           norm_path = norm_path.toUpperCase();
+        }
```

Comment:
This is an incorrect assumption. Whether case-sensitivity is enabled depends not only on the platform but also on the filesystem used (one may have mounted a Linux filesystem) and **even on which folder the file is in** because on NTFS you can enable case-sensitivity per folder (I have it enabled on my projects folder, for example, because I want to catch capitalization errors early before they crash on a Linux server later on)!